HW_06.Task_02/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace HW_06.Task_02
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] RandomArray = new int[3];
```

Comment:
incorrect naming for local variable - RandomArray, UserArray, NewNum, PosNum , etc.

Should be `randomArray, userArray, newNum, posNum`